src/sentry/static/sentry/app/views/performance/transactionDetails/quickTrace.tsx

```diff
@@ -258,12 +256,7 @@ function EventNodeDropdown({
       {events.length === 0
         ? null
         : events.slice(0, numEvents).map((event, i) => {
-            const target = generateSingleEventTarget(
-              event,
-              organization,
-              projects,
-              location
-            );
+            const target = generateSingleEventTarget(event, organization, location);
```

Comment:
@Zylphrex it looks like `withProjects()` can be removed entirely from this file, along with the `projects` prop.

Reply:
Good catch. Removed in e96edaa